src/Microsoft.Health.Dicom.Core/Features/Query/QueryResponseBuilder.cs

```diff
@@ -15,7 +16,7 @@ namespace Microsoft.Health.Dicom.Core.Features.Query;
 
 internal class QueryResponseBuilder
 {
-    internal static readonly HashSet<DicomTag> DefaultStudyTags = new HashSet<DicomTag>()
+    internal static readonly ImmutableHashSet<DicomTag> DefaultStudyTags = ImmutableHashSet.CreateRange(new DicomTag[]
```

Comment:
The only new change is that I've changed these to `ImmutableHashSet<T>` objects (not frozen ones as they're multi-targeted for .NET 6 too).